config/filesystem.json

```diff
@@ -13,7 +13,8 @@
 			{"type" : "lod", "path" : "Data/h3abp_bm.lod"}, // Localized versions only, contains H3:AB patch data
 			{"type" : "lod", "path" : "Data/H3bitmap.lod"}, // Contains H3:SoD data (overrides H3:AB data)
 			{"type" : "lod", "path" : "Data/H3pbitma.lod"}, // Localized versions only, contains H3:SoD patch data
-			{"type" : "dir",  "path" : "Data"}
+			{"type" : "dir",  "path" : "Data"},
+			{"type" : "dir",  "path" : "HcData"} // Chronicles
```

Comment:
I don't know original structure of chronicles, but will it work if player just want to use only resources from chronicles to play vcmi? It has almost all assets

Reply:
Yes, but:
- I think every chronicles user has also h3 complete
- It's really complicated (some assets are missing; some are replaced)
- We need a renaming or separate namespaces. To allow to load multiple Chronicles (use same assets-names for other assets)

Goal should be a running H3 with the chronicles as extension, not just running one (chronicle/complete) at time.